### DISCUSSION TITLE
starred count issue

### DISCUSSION THREAD

**Discussion by diamant3** [community/community#17293] [Mobile]
Inaccurate counting of stars

sample:

Mobile version:
![Screenshot_20220526-140931.png](https://user-images.githubusercontent.com/71203851/170428146-58cd731d-a062-491b-8fb4-e94ae6ed3f83.png)

Desktop version:
![Screenshot_20220526-141038.png](https://user-images.githubusercontent.com/71203851/170428215-9ec188d3-f140-4082-a8dc-5ca8bba22ac4.png)

**Comment by brianlovin** [comment]
Thanks for the report! I'll flag this for our team to investigate.

**Reply by brianlovin** [reply]
@diamant3 — we think that this _might_ be related to SAML-authorized organizations. Is it possible that when signing in to the GitHub Mobile app, not all of your SAML-gated organizations where authorized? It could be the case that you've starred repositories that are in that protected organization, which the website knows you have access to, but the mobile apps don't.

**Reply by diamant3** [reply]
thanks, is that a bug in mobile app or related to organization's repo i starred?

**Reply by brianlovin** [reply]
It's a bit hard to say if it's even a bug. What you could do to test is:

- sign out of the Mobile app
- sign out of GitHub.com in your primary browser
- sign back in to the Mobile app, and be sure to tap on all the green `Authorize` buttons that appear on the SAML-authorization step of the sign-in
- check to see if the star count is correct

**Reply by diamant3** [reply]
I tried that, but it didn't work, maybe I'll just wait for other people who are having the same problem as me, thanks!

latest sample 

Mobile version:

![Screenshot_20220601-092049.png](https://user-images.githubusercontent.com/71203851/171309560-fc433a15-9780-4b9e-aa03-678ccb1886f8.png)

Desktop version:

![IMG_20220601_091936_7.jpg](https://user-images.githubusercontent.com/71203851/171309600-1d2415b5-fb55-40c1-9f38-bdf9ffe5a538.jpg)

**Comment by diamant3** [comment]
My guess is that this issue is about "starred topics" because it didn't count those topics as stars.
